apps/portal/src/content/docs/components/drawer.mdx

```diff
@@ -4,57 +4,192 @@ description: Drawer component with stacking
+            Description
+          </Drawer.Description>
+        </Drawer.Header>
+
+        <Drawer.Body>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur at sagittis justo.
+            Integer vehicula turpis at justo gravida, at convallis lorem tincidunt.
+          </p>
+          <p>
+            Suspendisse potenti. Praesent in vestibulum lacus. Cras feugiat eros at ex cursus,
+            vel consequat enim mattis. Quisque nec magna ac leo tempor scelerisque.
+          </p>
+          <p>
+            Nullam tristique, justo vitae laoreet pretium, nisl nunc facilisis nulla, non porta erat sapien in justo.
+            Sed nec massa nec turpis interdum volutpat. Morbi sed justo ac leo sagittis tincidunt.
+          </p>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur at sagittis justo.
+            Integer vehicula turpis at justo gravida, at convallis lorem tincidunt.
+          </p>
+          <p>
+            Suspendisse potenti. Praesent in vestibulum lacus. Cras feugiat eros at ex cursus,
+            vel consequat enim mattis. Quisque nec magna ac leo tempor scelerisque.
+          </p>
+          <p>
+            Nullam tristique, justo vitae laoreet pretium, nisl nunc facilisis nulla, non porta erat sapien in justo.
+            Sed nec massa nec turpis interdum volutpat. Morbi sed justo ac leo sagittis tincidunt.
+          </p>
+          <p>
+            Suspendisse potenti. Praesent in vestibulum lacus. Cras feugiat eros at ex cursus,
+            vel consequat enim mattis. Quisque nec magna ac leo tempor scelerisque.
+          </p>
+          <p>
+            Nullam tristique, justo vitae laoreet pretium, nisl nunc facilisis nulla, non porta erat sapien in justo.
+            Sed nec massa nec turpis interdum volutpat. Morbi sed justo ac leo sagittis tincidunt.
+          </p>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur at sagittis justo.
+            Integer vehicula turpis at justo gravida, at convallis lorem tincidunt.
+          </p>
+          <p>
+            Suspendisse potenti. Praesent in vestibulum lacus. Cras feugiat eros at ex cursus,
+            vel consequat enim mattis. Quisque nec magna ac leo tempor scelerisque.
+          </p>
+          <p>
+            Nullam tristique, justo vitae laoreet pretium, nisl nunc facilisis nulla, non porta erat sapien in justo.
+            Sed nec massa nec turpis interdum volutpat. Morbi sed justo ac leo sagittis tincidunt.
+          </p>
+        </Drawer.Body>
+
+        <Drawer.Footer>
+          <ButtonGroup>
+            <Drawer.Close asChild>
+              <Button variant="outline">Cancel</Button>
+            </Drawer.Close>
+            <Button>Action</Button>
+          </ButtonGroup>
+        </Drawer.Footer>
+      </Drawer.Content>
+    </Drawer.Root>
+
+`}
+/>
+
+<DocsPage.ComponentExample
```

Comment:
Can we take this example and create a new section below "Anatomy", perhaps called "Nested drawers" to explain the concept and implementation. Would be nice to have the same for size and direction too.